our-stack.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Our Stack
+description: These are the programming languages, libraries, frameworks and tools we work with
+---
+
+### Javascript
+
+The frontend of most of our web apps are written in Javascript. We are using more and more Typescript to reduce bugs. The libraries [React](https://reactjs.org/), [Redux](https://redux.js.org/) and [Redux-Saga](https://github.com/redux-saga/redux-saga) are helping us to creating these web apps.
+
+For learning React, we have no special (free) resource to recommend. [A search engine can be of help here](https://duckduckgo.com/?q=learn+react). Wes Bos provides [an awesome free Redux course](https://learnredux.com/). He also has a paid React course. In order to learn Redux-Saga, [the docs of Redux-Saga](https://redux-saga.js.org/) are a really good starting point. Knowing Redux already might be of help before reading the Redux-Saga docs.
+
+
+### Ruby
+
+Most of our backend is written in [Ruby](https://www.ruby-lang.org/en/) and uses [Ruby on Rails](http://rubyonrails.org/). To get a first impression of Ruby, [Try Ruby](https://tryruby.org) should be your first stop. Other than that, [CodeCademy provides a free Ruby course](https://www.codecademy.com/learn/learn-ruby) to help you getting started. For Ruby on Rails we recommend the [Rails book](https://www.railstutorial.org/book).
+
+
+### Rust
+
+Meanwhile, we use a little bit of [Rust](https://www.rust-lang.org/en-US/) in the backend. We wrote a small PDF generation service called [pape-rs](https://github.com/store2be/pape-rs) with it. The [Rust Book](https://doc.rust-lang.org/book/) can provide you with the first steps in Rust.
```

Comment:
I don't think _meanwhile_ can be used like this in English, I think maybe you meant something like the German _mittlerweile_? A better option could be to use _recently we've been using a little bit of Rust_ or to drop it altogether.